crates/db_schema/src/aliases.rs

```diff
@@ -0,0 +1,3 @@
+use crate::schema::person;
+
+diesel::alias!(person as person1: Person1, person as person2: Person2);
```

Comment:
No need for a separate file, you should be able to put the cfg attribute directly on the macro. Or otherwise put a `mod {}` declaration in lib.rs